src/stp/plays/PenaltyUsPrepare.cpp

```diff
@@ -33,8 +33,9 @@ PenaltyUsPrepare::PenaltyUsPrepare() : Play() {
 uint8_t PenaltyUsPrepare::score(world::World* world) noexcept { return 100; }
 
 void PenaltyUsPrepare::calculateInfoForRoles() noexcept {
-    const auto xPosition = -4 * control_constants::ROBOT_RADIUS;
-    const auto yPosition = Constants::STD_TIMEOUT_TO_TOP() ? field.getFieldWidth() / 2.2 : -field.getFieldWidth() / 2.2;
+    const double xPosition = -4 * control_constants::ROBOT_RADIUS;
+    const double distanceToCenterLine = field.getFieldWidth() / 2 - 2*control_constants::ROBOT_RADIUS;
```

Comment:
Is this supposed to put them in a horizontal line above or below the x-axis of the field? Curious to know what the plan is

Reply:
Each robot's y-position is negative half the field, minus the robot diameter. So the middle point of the robots is one robot diameter into the field from the sideline at the "bottom" of the field.